src/main/java/eu/enmeshed/model/event/WebhookTrigger.java

```diff
@@ -32,7 +32,7 @@ public enum WebhookTrigger {
 
   @JsonProperty(
       WebhookTriggerNames.Consumption
-          .OUTGOING_REQUEST_FROM_RELATIONSHIP_CREATION_CHANGE_CREATED_AND_COMPLETED)
+          .OUTGOING_REQUEST_FROM_RELATIONSHIP_CREATION_CREATED_AND_COMPLETED)
   CONSUMPTION__OUTGOING_REQUEST_FROM_RELATIONSHIP_CREATION_CHANGE_CREATED_AND_COMPLETED,
```

Comment:
`CONSUMPTION__OUTGOING_REQUEST_FROM_RELATIONSHIP_CREATION_CHANGE_CREATED_AND_COMPLETED` must be replaced with `CONSUMPTION__OUTGOING_REQUEST_FROM_RELATIONSHIP_CREATION_CREATED_AND_COMPLETED` as well.

Reply:
Done